### PR TITLE
Civicrm

### PR DESCRIPTION
**Pull request by marcelklehr**

### CODE REVIEW

File: group_vars/sog_de/secret_credentials.yml [hunk 1]
```diff
@@ -1,60 +1,62 @@
...
-39353933383861303830376536326330646138646636373236376464643366313763643134663465
-64386465313538346261363661623735663266303565656662323036376235343962383536626162
-61636539613265666664353131656263363965313161333034356232643563303639646432386261
-30333461643337333363643166656665616536343462393031393432616135346633623331376134
-33663465323530363030613166353431616534643837663135626634353661396135383337666335
-39643330656563656335663437353035656632343862316331643232366166306433633631353134
-33623039333131333064363630633865383332383934623335393566613334613831623232346434
-62383830643463396130616536663761633564623634366161393965643063353465363234383136
-316230653362313862336666343231613731
+34323332613262653566336363306530376335653139313762323866636266366263646432643161
```
Comment (marcelklehr) [inline group_vars/sog_de/secret_credentials.yml]: this sucks when using branches :/

Comment (zonradkuse) [reply group_vars/sog_de/secret_credentials.yml]: I guess, easiest would be opening both and merging by hand :/

File: group_vars/sog_de/mysql.yml [hunk 2]
```diff
@@ -23,7 +24,10 @@ mysql_users:
      host: "%"
      password: "{{nextcloud_mysql_password}}"
      priv: "{{nextcloud_mysql_db}}.*:ALL"
-
+   - name: "{{civicrm_mysql_user}}"
+     host: "%"
+     password: "{{civicrm_mysql_password}}"
+     priv: "{{civicrm_drupal_mysql_db}}.*:ALL/{{civicrm_drupal_mysql_db}}.*:ALL"
```
Comment (zonradkuse) [inline group_vars/sog_de/mysql.yml]: Isn't it the same privilege listed twice?

File: site.yml [hunk 3]
```diff
@@ -8,8 +8,8 @@
     #- geerlingguy.git
     - php
     #- centos-common
-    #- geerlingguy.composer
-    #- acme-nginx
+    - geerlingguy.composer
+    - acme-nginx
     #- tomcat
```
Comment (zonradkuse) [inline site.yml]: It is tempting to do this but ansible provides some help here you can specify for example
`--start-at-task=`. As an alternative you can just spin up a new vserver deploying only the components you want.

**Comment by zonradkuse** [comment]
Supporting https://docs.civicrm.org/sysadmin/en/latest/upgrade/drupal7/ instead of a reinstall would be awesome!

File: site.yml (hunk 3 above)
Comment (zonradkuse) [reply site.yml]: Ha, even better. we should tag everything and then we can run only tagged tasks. We will need this anyway some time in the future.

File: group_vars/sog_de/mysql.yml (hunk 2 above)
Comment (marcelklehr) [reply group_vars/sog_de/mysql.yml]: that's an old diff ;)

File: site.yml (hunk 3 above)
Comment (marcelklehr) [reply site.yml]: tags sound nice!

File: group_vars/sog_de/mysql.yml (hunk 2 above)
Comment (zonradkuse) [reply group_vars/sog_de/mysql.yml]: ups :)

**Comment by marcelklehr** [comment]
> Supporting https://docs.civicrm.org/sysadmin/en/latest/upgrade/drupal7/ instead of a reinstall would be awesome!

I'm not so sure about how to go about this. Judging from their description (especially the prerequisites for upgrading), it doesn't sound like something one would lightly have done automatically, with all the things that could go wrong. I'd love having that, though.

**Comment by zonradkuse** [comment]
It could be fine if the revert to the previous version is also automated. But I see that this could be a lot of work.



On Sun, Mar 3, 2019 at 7:21 PM +0100, "Marcel Klehr" <notifications@github.com<mailto:notifications@github.com>> wrote:


Supporting https://docs.civicrm.org/sysadmin/en/latest/upgrade/drupal7/ instead of a reinstall would be awesome!

I'm not so sure about how to go about this. Judging from their description (especially the prerequisites for upgrading), it doesn't sound like something one would lightly have done automatically, with all the things that could go wrong. I'd love having that, though.

**Comment by marcelklehr** [comment]
It seems none of our civi extensions is compatible with v5.11.0 :/

**Pull request merged**